### DISCUSSION TITLE
Welcome to AEP Streaming Connect Discussions!

### DISCUSSION THREAD

**Discussion by nipunnair** [adobe/experience-platform-streaming-connect#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shole-habibi** [comment]
Hello,

I have one DataFlow ID and today there is one dataset ID is configured with it by  "aep.connection.endpoint.headers": "{\"x-adobe-flow-id\":\"7c076e06-d174-4la7-adf0-aa49a906f20b\"}", 
But I see in the developer guide documentation that we can have "aep.connection.endpoint.headers": "{\"x-adobe-flow-id\":\"341fd4f0-cdec-4912-1ab6-fb54aeb41286\", \"x-adobe-dataset-id\": \"3096fbfd5978431948af3ba3\"}"

I am in a usecase which I need to set two Dataset Ids to one dataflow Id. is there any example to show how we can do that?
I went through the code and seems it can support:
 "aep.connection.endpoint.headers": "{\"x-adobe-flow-id\":\"341fd4f0-cdec-4912-1ab6-fb54aeb41286\", \"x-adobe-dataset-id\": \"3096fbfd5978431948af3ba3\", \"3096fbfd5978431568af3ba8\"}". is that correct?

thanks,
